src/Gallery/Images.tsx

```diff
@@ -2,17 +2,20 @@ import React, { FC } from 'react';
 import { WhpptImageUploader } from '../ui/components/ImageUploader';
 import { WhpptGalleryImage } from '../ui/components/GalleryImage';
 import { GalleryTab } from './GalleryTab';
+import { useWhppt } from '..';
+
+export const Images: FC<GalleryTab> = ({ items, upload, setSelected, selectedId }) => {
+  const { domain } = useWhppt();
 
-export const Images: FC<GalleryTab> = ({ items, upload, setSelected, selectedId, domainId }) => {
   const getImgUrl = imgId => {
-    return `${process.env.NEXT_PUBLIC_BASE_API_URL}/gallery/image/${imgId}`;
+    return `${process.env.NEXT_PUBLIC_BASE_API_URL}/gallery/image/${imgId}?w=360`;
```

Comment:
Could we use the buildCroppedImgUrl function here to keep the logic in one place